content/4_Gold/Sliding_Window.mdx

```diff
@@ -264,6 +264,29 @@ static ArrayList<Integer> maxSlidingWindow(int[] nums, int k) {
 
 </JavaSection>
 
+<PySection>
+
+```python
+def maxSlidingWindow(self, nums: List[int], k: int) -> List[int]:
+	d = collections.deque()
+	
+	def enqueue(i):
```

Comment:
type annotations for this function as well

Reply:
Thanks for taking a look. I just added type annotations.